qa/rpc-tests/fundrawtransaction.py

```diff
@@ -28,7 +28,10 @@ def setup_network(self, split=False):
 
     def run_test(self):
         print "Mining blocks..."
-        feeTolerance = Decimal(0.00000002) #if the fee's positive delta is higher than this value tests will fail, neg. delta always fail the tests
+
+        # if the fee's positive delta is higher than this value tests will fail, neg. delta always fail the tests
+        #            = 2 bytes * minRelayTxFeePerByte
+        feeTolerance = 2 * Decimal('0.00005000')/1000
```

Comment:
@jonasschnelli  @TheBlueMatt Can you elaborate? I may be missing something but the rpc tests should not fail by changing the relayFee. Right now the bitcoin coin selection can only terminate if there is a tolerance of at least 2 bytes. (I.e. the transaction fee is paying for two bytes more than the tx actually has.) Consequently the `feeTolerance` is 2 bytes times whatever_you_pay_per_byte.

Regardless, #6828 (or this PR, which includes #6828) should be merged independent of #6722 because reverting minRelayTxFee is not "an alternative" to replacing this outdated magic number.